uk_bin_collection/tests/test_common_functions.py

```diff
@@ -227,10 +227,10 @@ def mock_os_path_exists(path):
 
 
 def test_create_webdriver_local():
-    result = create_webdriver(None)
+    result = create_webdriver(None, False)
     assert result.name == "chrome"
 
 
 def test_create_webdriver_remote():
-    result = create_webdriver("http://selenium:4444")
+    result = create_webdriver("http://selenium:4444", False)
```

Comment:
@robbrad same as above here.

Reply:
@dannya - nice find

I would not hard code it false there however

I would do it like this https://github.com/robbrad/UKBinCollectionData/blob/96d16426329dea5fd596add44ffda7c53d0477b2/uk_bin_collection/uk_bin_collection/councils/CalderdaleCouncil.py#L38,L43

This allows for input from the test runner or other debug / development process